gravity-forms/gw-quantity-as-decimal.php

```diff
@@ -90,6 +93,114 @@ function modify_quantity_input_tag( $markup, $field, $value, $lead_id, $form_id
 		return $markup;
 	}
 
+	function modify_submission_data( $submission_data, $feed, $form, $entry ) {
```

Comment:
This method is pretty hefty, and I'm admittedly having a hard time following what exactly we're doing here.

The appears to be a copy and paste of `get_order_data` with some changes. Is that correct? If so, can you call that out and note what changes you made so if something changes upstream that breaks this snippet, we'll know what to update in the future?